cmd/root.go

```diff
@@ -81,6 +57,24 @@ func Execute(setupScripts *scripts.SetupScripts) {
 	fmt.Println("output: ", string(res))
 }
 
+func getCIProvider(args []string) (*string, error) {
+	if len(args) > 1 {
+		return &args[1], nil
+	}
+	ciProviders := []string{"Bitbucket", "CircleCI", "GitHub Actions", "GitLab CI", "Travis CI"}
```

Comment:
I'm new to `go` language. Can we not import/use the `const` defined in `scripts/scripts.go` to avoid repetition? If we can't that's totally fine.